pinot-server/src/main/java/org/apache/pinot/server/api/resources/TablesResource.java

```diff
@@ -610,7 +614,7 @@ public String getValidDocIdMetadata(
   public String getValidDocIdMetadata(
```

Comment:
Introduced in the previous PR. Can we change this to GET request? Maybe change the path to "/tables/{tableNameWithType}/validDocIdsMetadata"

Reply:
changed to `validDocIdMetadata -> calidDocIdsMetadata`